src/Host/Client/Impl/Host/BrokerClientProxy.cs

```diff
@@ -26,10 +26,10 @@ internal sealed class BrokerClientProxy : IBrokerClient {
             broker.Dispose();
         }
 
-        public string Name => _broker.Name;
-        public bool IsRemote => _broker.IsRemote;
-        public Uri Uri => _broker.Uri;
-        public AboutHost AboutHost => _broker.AboutHost;
+        public string Name => _broker?.Name;
```

Comment:
Specifically in 

```
        private void OnSessionConnectionStateChange(bool connected) {
            IsConnected = connected;
            UpdateActiveConnection();
            ConnectionStateChanged?.Invoke(this, new ConnectionEventArgs(IsConnected, ActiveConnection));
        }
```

which technically may come, i think, after broker was removed